rust/operator-binary/src/crd.rs

```diff
@@ -47,6 +47,14 @@ pub enum SecretClassBackend {
     /// A new certificate and keypair will be generated and signed for each Pod, keys or certificates are never reused.
     AutoTls(AutoTlsBackend),
 
+    /// The [`experimentalCertManager` backend](DOCS_BASE_URL_PLACEHOLDER/secret-operator/secretclass#backend-certmanager)
+    /// injects a TLS certificate issued by [cert-manager](https://cert-manager.io/).
+    ///
+    /// A new certificate will be requested the first time it is used by a Pod, it will be reused after that (subject to
+    /// cert-manager renewal rules).
```

Comment:
**nit (non-blocker)**: I think we should move links to the bottom (named or numbered, I don't mind).

```suggestion
    /// The [`experimentalCertManager` backend][1] injects a TLS certificate issued
    /// by [cert-manager](https://cert-manager.io/).
    ///
    /// A new certificate will be requested the first time it is used by a Pod, it
    /// will be reused after that (subject to cert-manager renewal rules).
    ///
    /// [1]: DOCS_BASE_URL_PLACEHOLDER/secret-operator/secretclass#backend-certmanager
```

Reply:
Makes sense for at least those massive links, agreed...